tools/monorepo-merge/src/commands/transfer-issues/index.ts

```diff
@@ -167,10 +167,7 @@ export default class TransferIssues extends Command {
 				continue;
 			}
 
-			this.resetProjectFields(
-				authenticatedGraphQL,
-				issue
-			);
+			this.resetProjectFields( authenticatedGraphQL, issue );
```

Comment:
Based on the PR description, the changes to this file look accidental. Is this from another PR?